### PR TITLE
Germany: Fix Frankfurt records

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1294.

PR includes the following edits:

- Updated `hasc:id` concordances
- Updated `name:eng_x_preferred` values (including fixing trailing spaces)
- Swapped `statoids:*` properties
- Updated `wof:population` and `wof:population_rank` properties

All of these issues were a result of joining Statoids data to the incorrect Frankfurt records.

### CODE REVIEW

File: data/102/063/919/102063919.geojson [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "mz:hierarchy_label":1,
     "mz:is_current":1,
     "name:eng_x_preferred":[
-        "Frankfurt am Main "
+        "Frankfurt (Oder)"
```
Comment (nvkelso) [inline data/102/063/919/102063919.geojson]: Should the preferred English name just be `Frankfurt` and `Frankfurt am Main` as a variant? May also be helpful to store the longer disambiguation of `Frankfurt am Main` in **label**?

File: data/102/063/589/102063589.geojson [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "mz:hierarchy_label":1,
     "mz:is_current":1,
     "name:eng_x_preferred":[
-        "Frankfurt (Oder) "
+        "Frankfurt am Main"
```
Comment (nvkelso) [inline data/102/063/589/102063589.geojson]: Should the preferred English name just be `Frankfurt` and `Frankfurt am Main` as a variant? May also be helpful to store the longer disambiguation of `Frankfurt am Main` in **label**?

File: data/102/063/919/102063919.geojson [hunk 3]
```diff
@@ -69,13 +69,13 @@
         }
     ],
     "wof:id":102063919,
-    "wof:lastmodified":1527637204,
+    "wof:lastmodified":1536081746,
     "wof:name":"Frankfurt (Oder)",
```
Comment (nvkelso) [inline data/102/063/919/102063919.geojson]: Should the WOF name just be `Frankfurt`?

File: data/102/063/589/102063589.geojson [hunk 4]
```diff
@@ -71,13 +71,13 @@
         }
     ],
     "wof:id":102063589,
-    "wof:lastmodified":1505430535,
+    "wof:lastmodified":1536081743,
     "wof:name":"Frankfurt am Main",
```
Comment (nvkelso) [inline data/102/063/589/102063589.geojson]: Should the WOF name just be `Frankfurt`?

File: data/102/063/919/102063919.geojson (hunk 1 above)
Comment (nvkelso) [reply data/102/063/919/102063919.geojson]: Should also have variant name of `Frankfurt an der Oder`

File: data/102/063/589/102063589.geojson (hunk 4 above)
Comment (stepps00) [reply data/102/063/589/102063589.geojson]: That works, as long as this name is saved as a variant..

Other names:
- City of Frankfurt am Main
- Frankfurt on the Main
- Frankfurt am Main

https://en.wikipedia.org/wiki/Frankfurt

File: data/102/063/919/102063919.geojson (hunk 1 above)
Comment (stepps00) [reply data/102/063/919/102063919.geojson]: Yep.. https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/label

Comment (nvkelso) [reply data/102/063/919/102063919.geojson]: Let's also at least add the Germany names and labels in this PR.

File: data/102/063/589/102063589.geojson (hunk 2 above)
Comment (nvkelso) [reply data/102/063/589/102063589.geojson]: Let's also at least add the Germany names and labels in this PR.

**Comment by stepps00** [comment]
Took care of reviewer comments in https://github.com/whosonfirst-data/whosonfirst-data/pull/1298/commits/584ed8cf789f30d2bed7188a46c77cc116aa3d88.

PR should be ready to merge.

**Pull request merged**